alfresco-connector-test/src/alfresco/ecm/connector/test/AlfrescoMock.java

```diff
@@ -9,26 +9,35 @@
 import javax.ws.rs.Path;
 import javax.ws.rs.Produces;
 import javax.ws.rs.core.MediaType;
+
 import org.apache.commons.io.IOUtils;
+
 import io.swagger.v3.oas.annotations.Hidden;
 
 /**
  * Mocks parts of the Alfresco ECM Rest API which has been used in the AlfrescoEcmDemo
  * <p>docs<br/>https://api-explorer.alfresco.com/api-explorer/definitions/alfresco-core.json</p>
  */
-@Path("ecmMock")
+@Path(AlfrescoMock.PATH_SUFFIX)
 @PermitAll
 @Hidden
 public class AlfrescoMock
-{ 
+{
+  static final String PATH_SUFFIX = "ecmMock";
+
+  // URI where this mock can be reached: to be referenced in tests that use it!
+  @SuppressWarnings("restriction")
+  public static final String URI = "{"+ch.ivyteam.ivy.rest.client.config.IvyDefaultJaxRsTemplates.APP_URL+"}/api/"+PATH_SUFFIX;
```

Comment:
imo: it would be more readable to directly use the string. because the string is public api and must not be changed, wheres IvyDefaultJaxRsTemplates is not public api.